k3k-kubelet/kubelet.go

```diff
@@ -71,15 +78,51 @@ func newKubelet(ctx context.Context, c *config, logger *k3klog.Logger) (*kubelet
 	}
 
 	virtClient, err := kubernetes.NewForConfig(virtConfig)
+	if err != nil {
+		return nil, err
+	}
+
+	hostMgr, err := ctrl.NewManager(hostConfig, manager.Options{
+		Scheme: baseScheme,
+		Metrics: ctrlserver.Options{
+			BindAddress: ":8083",
```

Comment:
do we need the metrics here?

Reply:
I figure that it can't hurt. I can disable it if you think that's necessary, but I think that it makes sense to leave that on unless we have a reason.

Reply:
makes sense